models/Discord/Commands/DiscordCommandMentor.js

```diff
@@ -0,0 +1,44 @@
+const DiscordCommand = require('../DiscordCommand.js');
+
+class DiscordCommandMentor extends DiscordCommand {
+
+    constructor(subsystem) {
+        super("mentor", "Give/remove a mentor's rank", 'mentor', subsystem);
+    }
+
+        onRun(message, permissions, args) {
+        var config = this.subsystem.manager.getSubsystem("Config").config;
+        if (args.length < 2) {
+            message.reply("Usage is `" + config.discord_command_character + "addao <add/remove> [ckey]`");
+            return;
+        }
+        var removing = args[0];
+        var ckey = args[1];
+
+        var dbSubsystem = this.subsystem.manager.getSubsystem("Database");
+        dbSubsystem.pool.getConnection((err, connection) => {
+            connection.query('SELECT * FROM `erro_mentor` WHERE `ckey` = ?', [ckey], (error, results, fields) => {
+                if(results.length > 0) {
+                    message.reply("Player is already a mentor.");
+                    return;
```

Comment:
Unnecessary return.

Reply:
what do you want me to replace it with?

Reply:
The way your code is at the moment you should replace it with nothing... Look at my comment below this one as you might need to keep it depending on how you make your code work.